### ISSUE TITLE
Lock Screen for battery saving

### ISSUE THREAD

**Issue by woogieman007** [Deletescape-Media/Lawnchair#939]
Would love to see this app add a lock screen tool which would help save battery life. Sometimes phones like mine the screen will switch it's self on in your pocket and start doing random stuff but a screen lock like what Power+Launcher-Battery Saver has would save battery and prevent problems like that from happening.  To me battery usage of an app would be top of my list rather then a fancy interface. PS love the app so far. Keep up the good work

**Comment by divadsn** [comment]
This isn't our job to do that, this has to be done by the ROM (see Pocket Lock on several Nougat custom ROMs) or an accessibility service, which would end for us badly as such apps are now getting banned on the Play Store.

Tl;dr: It's unlikely that we will add such a feature.

**Comment by woogieman007** [comment]
Just gotta hope this app uses low battery use as that's more impotent to me, if I can find an app like the other launcher 's lock screen then this app is well worth keeping

**Comment by guest4711** [comment]
Try WaveUp (Turn the display on by waving) - https://f-droid.org/app/com.jarsilio.android.waveup .
